src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Iconography/IconographyPage.razor.scss

```
@@ -4,18 +4,122 @@
-        cursor: copy;
+        width: 5.125rem;
+        cursor: pointer;
+        margin: 1px;
         padding: 0.5rem;
+        border-radius: 0.25rem;
+        transition: background-color 0.15s ease;
 
         .icon-name {
             opacity: 0;
+            text-align: center;
+            word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)